lib/wasix/Cargo.toml

```diff
@@ -49,6 +49,7 @@ tokio = { version = "1", features = [
 ], default_features = false }
 tokio-stream = { version = "0.1", features = [ "sync" ] }
 futures = { version = "0.3" }
+async-ctrlc = { version = "1.2.0", features = [ "stream" ], optional = true }
```

Comment:
Tokio has built-in support for this: https://docs.rs/tokio/latest/tokio/signal/fn.ctrl_c.html